libs/single-spa-angular/package.json

```diff
@@ -24,8 +24,9 @@
   "homepage": "https://github.com/single-spa/single-spa-angular#readme",
   "sideEffects": false,
   "peerDependencies": {
-    "@angular/core": ">=12",
+    "@angular/core": ">=13",
```

Comment:
This change likely requires that this pull request be published as a new breaking change in single-spa-angular@6 (which is okay)